infrastructure/modules/dynamo_db/main.tf

```diff
@@ -65,9 +56,12 @@ resource "aws_iam_policy" "dynamodb_policy" {
       {
         "Effect" : "Allow",
         "Action" : [
+          "dynamodb:Query",
+          "dynamodb:Scan",
```

Comment:
I believe Scan is resource intensive. Is this needed, or will query do the same job?

Reply:
We checked that we are not using scan anywhere in Auth lambdas. This seems to be something we copied from ARF when we migrated the terraform code. 
We have removed the dynamodb:Scan permission now